Makefile

```diff
@@ -59,7 +59,8 @@ test: prometheus manifests generate fmt vet envtest ## Run tests with Prometheus
 	@echo "Starting Prometheus for testing..."
 	@$(PROMETHEUS) --config.file=pkg/testconfig/prometheus.yml & \
 	trap 'killall prometheus; rm -rf data' EXIT; \
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" ginkgo run -p 1 ./... --coverprofile cover.out -test.v --procs 1 --vv --output-interceptor-mode=none
+    KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" ginkgo run --cover --coverprofile cover.out ${OTTOSCALR_TEST_ARGS} --procs 1 --nodes 1 -v --output-interceptor-mode=none ./... -test.v
```

Comment:
you can remove the -test.v flag at the end

Reply:
done